app/src/services/mlService.ts

```
@@ -692,20 +692,19 @@ export const useGestureClassifier = (
   const enqueueFrameJS = createRunOnJS(enqueueFrame);
   const logErrorJS = createRunOnJS(logger.error);
   const onErrorJS = createRunOnJS((message: string) => onErrorRef.current?.(message));
-  const extractLandmarks = useHandLandmarkExtractor();
+  const extractLandmarks = pluginAvailable
+    ? useHandLandmarkExtractor()
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)